src/hooks/users.ts

```diff
@@ -1,52 +1,61 @@
 import { useMemo } from "react";
-import { useFirestoreConnect } from "react-redux-firebase";
+import { isLoaded } from "react-redux-firebase";
 
 import { usersSelector, usersByIdSelector } from "utils/selectors";
 import { WithId } from "utils/id";
 
 import { useSelector } from "hooks/useSelector";
 import { useUserLastSeenThreshold } from "hooks/useUserLastSeenThreshold";
 import { useVenueId } from "hooks/useVenueId";
+import { useSparkleFirestoreConnect } from "hooks/useSparkleFirestoreConnect";
 
 import { User } from "types/User";
 
 const useConnectUsers = () => {
-  useFirestoreConnect("users");
+  const venueId = useVenueId();
+
+  useSparkleFirestoreConnect([
+    {
+      collection: "users",
+      where: ["enteredVenueIds", "array-contains", venueId],
+      storeAs: "users",
```

Comment:
I think we would be better off calling this `venueUsers` or similar + update the related selector, to differentiate it from what the library calls it when we just do a raw 'collection' connection (the default for that is to use the name of the collection as the `storeAs` value)

Reply:
Created an issue for that
https://github.com/sparkletown/internal-sparkle-issues/issues/124

Reply:
Thanks :)